File: examples/java/kora-java-crud-submodule/kora-java-crud-submodule-vet-api/src/main/java/io/koraframework/example/submodule/vet/VetModule.java
package io.koraframework.example.submodule.vet;

import io.koraframework.cache.caffeine.CaffeineCacheModule;
import io.koraframework.common.annotation.KoraSubmodule;
import io.koraframework.database.jdbc.JdbcDatabaseModule;
import io.koraframework.example.submodule.common.CommonModule;
import io.koraframework.resilient.ResilientModule;

@KoraSubmodule
public interface VetModule extends
        CommonModule,
        JdbcDatabaseModule,
        CaffeineCacheModule,
        ResilientModule {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-vet-api/src/main/kotlin/io/koraframework/kotlin/example/submodule/vet/VetModule.kt
package io.koraframework.kotlin.example.submodule.vet

import io.koraframework.cache.caffeine.CaffeineCacheModule
import io.koraframework.common.annotation.KoraSubmodule
import io.koraframework.database.jdbc.JdbcDatabaseModule
import io.koraframework.kotlin.example.submodule.common.CommonModule
import io.koraframework.resilient.ResilientModule

@KoraSubmodule
interface VetModule : CommonModule, JdbcDatabaseModule, CaffeineCacheModule, ResilientModule
